packages/web-components/src/components/cbp-action-bar/cbp-action-bar.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * @prop --cbp-action-bar-color-background: var(--cbp-color-grey-cool-70);
+ * @prop--cbp-action-bar-color-background-dark: var(--cbp-color-grey-cool-5);
+*/
+
+:root {
+ --cbp-action-bar-color-background: var(--cbp-color-gray-cool-70);
+ --cbp-action-bar-color-background-dark: var(--cbp-color-gray-cool-5);
+
+ --cbp-action-bar-info-color: var(--cbp-color-text-lightest);
+ --cbp-action-bar-info-color-dark: var(--cbp-color-text-darkest);
+  
+}
+
+[data-cbp-theme=light] cbp-action-bar[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-action-bar:not([context=dark-inverts]):not([context=light-always]) {
+  --cbp-action-bar-color-background: var(--cbp-action-bar-color-background-dark);
+  --cbp-action-bar-info-color: var(--cbp-action-bar-info-color-dark);
+  
+}
+
+cbp-action-bar {
+  display: flex;
+  align-items: center;
+  justify-content: right;
+  height: var(--cbp-space-14x);
```

Comment:
this should be a min-height. we don't set fixed heights because they can break if text is resized or wraps.